services/cache-headers.spec.js

```diff
@@ -35,6 +35,11 @@ describe('Cache header functions', function() {
         serviceDefaultCacheLengthSeconds: 900,
         queryParams: { maxAge: 1000 },
       }).expect(1000)
+      given({
+        cacheHeaderConfig,
+        serviceDefaultCacheLengthSeconds: 900,
```

Comment:
Would it be feasible here to update the naming conventions used here (and in cache-headers.js`) to match with the new naming conventions used in the base service (basically dropping `Length` I believe)

Reply:
Yea, I dropped `Length` which doesn't seem to add anything (cache seconds == cache length seconds). Agreed that cleaning up the rest of them is a nice idea, though I'd rather deal with it in a follow-on. `BaseService` has a property called `_cacheLength` that probably should be renamed to `cacheSeconds` and that's bound to affect a handful of files and code that's spread about a few places.

Reply:
Gotcha and agreed 👍 When I first saw this I thought I was still up in the BaseService file, took me a second to remember I'd moved on to another file